tests/expected/arith_checks/fixme_check_arith_failures.rs

```diff
@@ -0,0 +1,13 @@
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+//! Check that code after an overflow check failure is unreachable.
```

Comment:
Perhaps change the comment to say: should be unreachable, and add a link to #1208?

Reply:
Indeed!